site/source/tutorials/008_animations/index.md

```diff
@@ -0,0 +1,370 @@
+---
+layout: tutorial
+title: Web Animations
+overview: Use the Dojo 2 WebAnimations meta to animate your widgets.
+paginate: true
+---
+
+{% section 'first' %}
+
+# Web Animations
+
+## Overview
+This tutorial introduces two lovable zombies which we will learn to animate using the Dojo 2 `WebAnimation` meta. We will introduce the api provided by WebAnimations and show you how to use it with Dojo 2.
```

Comment:
WebAnimations -> Web Animations, and perhaps should link to https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API ?